docs/userguide/userguide.md

```diff
@@ -1,6 +1,7 @@
 # User guide
 
 ```{toctree}
+Active Learning <active_learning>
```

Comment:
Should this really be the first user guide? Iirc we once discussed that Campaign should be the first.

Reply:
ok then Ill put it one down? I think anything below `Campaign` was alphabetical